client/my-sites/comments/comment/style.scss

```diff
@@ -23,6 +23,11 @@
 	}
 }
 
+// Hide comment tree replies bottom box-shadow by removing the margin
+.card.comment .card.comment {
```

Comment:
This is applying to all comments nested within comments, not just when we hit the max depth. Is that intentional?